Promotions. Everyone gets a medal

Inscrutable opened the pull request:
worm424 => Docs
phase => Ore Back-end Manager

Inscrutable committed:
Promotions

worm424 => Docs
phase => Ore Back-end Manager

Spongy commented:
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/a7844ce/.

Here are some links to the pages that were modified:

- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/a7844ce/about/staff.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/726#issuecomment-408583241), not to the direct url to the preview._

ST-DDT commented on source/about/staff.rst:
```diff
@@ -53,16 +52,17 @@ The **Project Managers** of Sponge each coordinate a separate aspect of the Spon
 | progwml6                          | Ore Back-end Manager               |
 |                                   |                                    |
-|                                   | Systems Management                 |
+|                                   | Systems Manager                    |
```
Maybe `System Manager`, but I'm not sure.

Inscrutable replied:
I'm just mirroring the phrasing from lukegb's position.

gabizou commented on source/about/staff.rst:
```diff
@@ -53,16 +52,17 @@ The **Project Managers** of Sponge each coordinate a separate aspect of the Spon
 |                                   |                                    |
 |                                   | Website Manager                    |
 +-----------------------------------+------------------------------------+
+| phase                             | Ore Back-end Manager               |
```
s/Ore Back-end Manager/ Ore Development Manager

gabizou replied:
And mbaxter = Ore Community Manager, right @Zidane ?

phase replied:
`Ore Development Manager`

phase replied:
`Ore Development Manager`

gabizou commented on source/about/staff.rst:
```diff
@@ -53,16 +52,17 @@ The **Project Managers** of Sponge each coordinate a separate aspect of the Spon
++-----------------------------------+------------------------------------+
 | phroa                             | IRC Manager                        |
 +-----------------------------------+------------------------------------+
 | progwml6                          | Ore Back-end Manager               |
```
nix the Ore Back-end Manager bit, since it's bundled with Systems Manager.

phase replied:
`Ore Review Team Manager`

phase reviewed (changes requested):
these titles are clear

phase commented:
oh wait I commented on the wrong person! mbax shall be the `Ore Review Team Manager`

Inscrutable committed:
Tweaking titles

mbaxter -> Ore Community Manager
phase -> Ore Development Manager 
progwml6 -> Systems Manager

phase commented:
Do we want to add the `Sysadmin` title to Snowie & kashike?

Zidane commented:
@phase Yes.

Inscrutable commented:
Is `Sysadmin` a new role, and if so, where does it fit (a new section below *project managers*)?
We already have two `Systems Manager`s. I'm starting to wonder what all these things really mean.

Inscrutable committed:
Ore reviewers added

This also gives kashike three roles; if we add Sysadmin that will be 4.
@mbax can veto or tinker with this PR if he so chooses.

ST-DDT commented:
A list of positions and their purpose would really be nice.

Inscrutable committed:
More promotions and renamings

Systems Manager -> System Administrator
except lukegb who is now Systems Team Manager
New section for Systems Administrators

Inscrutable committed:
Bolder

mbax reviewed (commented):
Great! I'm late, but I wanted to feel like I was a part of it!

Inscrutable commented:
Well, you *did* give me the list of Ore Reviewers to publish, so I figured that was approval.
